packages/shared/invokeGuardedCallback.js

```diff
@@ -173,6 +179,12 @@ if (__DEV__) {
         this._caughtError = null;
       }
 
+      if (shouldIgnoreError && error != null) {
+        this._hasSuppressedError = true;
+        this._suppressedError = error;
+        error.suppressReactErrorLogging = true;
```

Comment:
while running yarn flow I am getting error `property `suppressReactErrorLogging`. Property not found in Error`.